### PR TITLE
Remove mentions of nix-ld-rs

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Mic rewrote nix-ld and integrated the fork into it and that is the current default implementation in nixos-unstable

### CODE REVIEW

**Comment by nzbr** [comment]
But that would make the instructions fail for people on stable. It would be better to add a note that changing the package is not needed on unstable

**Comment by SuperSandro2000** [comment]
Why do we have stable support in the main branch...

File: docs/src/how-to/vscode.md [hunk 1]
```diff
@@ -18,14 +18,11 @@ environment.systemPackages = [
 [nix-ld](https://github.com/Mic92/nix-ld) is a program that provides `/lib64/ld-linux-x86-64.so.2`,
 allowing foreign binaries to run on NixOS.
 
-Running the VSCode server on NixOS-WSL requires using [nix-ld-rs](https://github.com/nix-community/nix-ld-rs), instead of the regular nix-ld.
```
Comment (SuperSandro2000) [inline docs/src/how-to/vscode.md:21]:
```suggstion
Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as often writing only on unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs).

```

File: docs/src/how-to/vscode.md [hunk 2]
```diff
@@ -18,14 +18,11 @@ environment.systemPackages = [
...
-
 To set it up, add the following to your configuration:
 
 ```nix
 programs.nix-ld = {
     enable = true;
-    package = pkgs.nix-ld-rs;
```
Comment (SuperSandro2000) [inline docs/src/how-to/vscode.md:28]:
```suggestion
      package = pkgs.nix-ld-rs; # only for NixOS 23.05
```

**Comment by SuperSandro2000** [comment]
changed it to mention only unstable

**Comment by nzbr** [comment]
Because I never got around to do something about #484

File: docs/src/how-to/vscode.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ environment.systemPackages = [
 [nix-ld](https://github.com/Mic92/nix-ld) is a program that provides `/lib64/ld-linux-x86-64.so.2`,
 allowing foreign binaries to run on NixOS.
 
-Running the VSCode server on NixOS-WSL requires using [nix-ld-rs](https://github.com/nix-community/nix-ld-rs), instead of the regular nix-ld.
+Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as often writing only on unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs).
```
Comment (nzbr) [inline docs/src/how-to/vscode.md:21]: I think something went wrong with this sentence. At least my brain refuses to parse it

Comment (SuperSandro2000) [reply docs/src/how-to/vscode.md:21]:
```suggestion
Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as of writing only on NixOS unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs) on NixOS 24.05.
```

File: docs/src/how-to/vscode.md [hunk 4]
```diff
@@ -18,14 +18,14 @@ environment.systemPackages = [
...
 
 To set it up, add the following to your configuration:
 
 ```nix
 programs.nix-ld = {
     enable = true;
-    package = pkgs.nix-ld-rs;
+    package = pkgs.nix-ld-rs; # only for NixOS 23.05
```
Comment (SuperSandro2000) [inline docs/src/how-to/vscode.md:28]:
```suggestion
    package = pkgs.nix-ld-rs; # only for NixOS 24.05
```

File: docs/src/how-to/vscode.md (hunk 3 above)
Comment (SuperSandro2000) [reply docs/src/how-to/vscode.md]: derp

**Pull request merged**